patch/driver-support-sff-8436-eeprom-update.patch

```diff
@@ -1,116 +1,71 @@
-Update SFF8436 EEPROM driver
+Support newer kernel and remove eeprom_class dependency in sff_8436_eeprom
```

Comment:
1.  Has this been sent upstream? If yes, please add the references.
2.  Please format the patch file with `git format-patch -1`, so it can easily be applied with `git am …`.